packages/dds/shared-object-base/src/types.ts

```diff
@@ -31,10 +31,10 @@ export interface ISharedObject<TEvent extends ISharedObjectEvents = ISharedObjec
     isAttached(): boolean;
 
     /**
-     * Generates summary of the shared object.
-     * @returns A tree representing the summary of the shared object.
+     * Capture the current content of the channel to be used to generate a summary
+     * @returns Object containing captured state that exposes functionality over that state (e.g. produce summary)
      */
-    summarize(fullTree?: boolean, trackState?: boolean): ISummaryTreeWithStats;
+    captureState(): ISnaphost;
```

Comment:
Maybe I'm blind, but I can't figure out what is this "ISnaphost"
I see only one file changed in that package (packages/runtime/datastore-definitions/src/channel.ts) and it's not there (and not in repo).
Also is it a typo (i.e. should be ISnapshot)?